features/functests/performance/performance.go

```diff
@@ -50,20 +62,160 @@ var _ = Describe("performance", func() {
 	var _ = Context("Pre boot tuning setup", func() {
 		It("Should contain a custome initrd image in boot loader", func() {
 			nodes, err := clients.K8s.CoreV1().Nodes().List(metav1.ListOptions{
-				LabelSelector: "node-role.kubernetes.io/worker-rt=",
+				LabelSelector: perfWorkerNodesLabel,
 			})
 			Expect(err).ToNot(HaveOccurred())
+			Expect(len(nodes.Items)).Should(BeNumerically(">", 0), "cannot find nodes labeled as "+perfWorkerNodesLabel)
+
 			for _, node := range nodes.Items {
 				mcd, err := mcdForNode(&node)
 				Expect(err).ToNot(HaveOccurred())
 				mcdName := mcd.ObjectMeta.Name
+				By("executing a command inside the pod " + mcdName)
 				bootLoaderEntries, err := exec.Command("oc", "rsh", "-n", "openshift-machine-config-operator", mcdName,
 					"grep", "-R", "initrd", "/rootfs/boot/loader/entries/").CombinedOutput()
 				Expect(err).ToNot(HaveOccurred())
-				Expect(strings.Contains(string(bootLoaderEntries), "iso_initrd.img")).To(BeTrue())
+				Expect(strings.Contains(string(bootLoaderEntries), "iso_initrd.img")).To(BeTrue(), "cannot find iso_initrd.img entry among the bootloader entries")
+			}
+		})
+	})
+
+	var _ = Context("Network latency parameters", func() {
+		It("Should contain configuration injected through the 00-tuned-network-latency template", func() {
```

Comment:
We are dependent on the tuned-network-latency profile here, for this repo it might be ok.
@cynepco3hahue  any thoughts here ?

Reply:
Our worker-rt tuned profile depends on it, so I think it should be ok